code/game/objects/structures/fence.dm

```diff
@@ -89,7 +90,10 @@
 	if(shock(user, 100))
 		return
 	if(!cuttable)
```

Comment:
I'd move this check below the `invulnerable` check. Else you can remove the corners of an `invulnerable` fence